docs/pages/development/installation.md

```diff
@@ -144,7 +128,7 @@ When you start your project on iOS, the metro bundler will be started automatica
 
 ### Add better error handlers
```

Comment:
This one in particular people should hopefully not be skipping

Reply:
I moved this one to the top, to indicate its importance.